homeassistant/components/matter/sensor.py

```diff
@@ -812,6 +878,38 @@ def _update_from_device(self) -> None:
             clusters.ElectricalPowerMeasurement.Attributes.ActiveCurrent,
         ),
     ),
+    MatterDiscoverySchema(
+        platform=Platform.SENSOR,
+        entity_description=MatterSensorEntityDescription(
+            key="ElectricalPowerMeasurementReactiveCurrent",
+            device_class=SensorDeviceClass.CURRENT,
```

Comment:
Should we set translation keys for the current sensors since there are more than one, so it's possible to distinguish them?

Reply:
Same question for the voltage sensors.